javascript/src/main/components/Schedule/AddToScheduleTable.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import { Button } from "react-bootstrap";
+
+const AddToScheduleTable = ({ data }) => {
+
+  const renderAddToButton = (id) => {
+     return (
+       <Button data-testid={`add-to-button-${id}`} onClick={() => {
+         
+       }}>Add To</Button>
```

Comment:
If the `onClick` function is empty, can we just remove it?

Reply:
We were planning on implementing the function later in another issue, but you're right, we can remove it until next issue